src/components/switch/__tests__/Switch.spec.js

```diff
@@ -0,0 +1,44 @@
+import Switch from "../Switch.svelte";
+import { render, clearContext } from "@/components/_util/testHelpers";
+import { tick } from "svelte";
+
+describe("Switch component", () => {
+  afterEach(() => {
+    clearContext();
```

Comment:
This gets auto added in the template generator. Sorry about that! Can you remove it?